### DISCUSSION TITLE
Inhaltsbasiertes Füllen

### DISCUSSION THREAD

**Discussion by Basnerwolfgang** [tannerhelland/PhotoDemon#610] [Q&A] [answered]
Ich möchte gerne auf einem Bild, das Gitter entfernen. Ist das möglich? Wenn ja, wie gehe ich vor.
![IMG_20241021_120716_DRO](https://github.com/user-attachments/assets/cbd5779e-5457-49c8-b3cc-0b4b9a74eebc)

**Comment by hi5** [comment, marked as answer]
Try:
- Magic Wand selection to select fence
- Selection, Grow by a few pixels to get some of the area outside the fence (just a few px)
- Edit, Content aware fill (ctrl-shift-F5), this is what I tried:
![settings](https://github.com/user-attachments/assets/61c61f81-b6fb-4451-8277-0a02424637cd)

Play around with some of the settings, quick & dirty example gives this result, pretty decent for a few seconds work:
![test](https://github.com/user-attachments/assets/460744db-1d81-4a48-906b-3fad2368c852)
